JobRunner/SpecialRunner.py

```diff
@@ -58,23 +62,27 @@ def _watch_batch(self, stype, job_id, slurm_jobid,
         self.logger.log("Watching Slurm Job ID %s" % (slurm_jobid))
         check = '%s_checkjob' % (stype)
         cont = True
-        started = False
+        retry = 0
```

Comment:
10 seconds of waiting with 5 retries? This looks like a very short time to let a job run. Is this intended?

Reply:
Oh wait, if its running, the retries are not incremented, never mind. Its 50 seconds of lookup failures?